### PR TITLE
[thomasmonkman-filewatch] Add patch for fixing Unix version

### PR DESCRIPTION
**Pull request by Pravila00**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] SHA512s are updated for each updated download
- [ ] The "supports" clause reflects platforms that may be fixed by this new version
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by Pravila00** [comment]
This patch comes from https://github.com/ThomasMonkman/filewatch/pull/29, which trigger events on Unix as it does on Windows for having the same behaviour in both platforms.

File: versions/t-/thomasmonkman-filewatch.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "09215796e0a4591e54d9aa37ab46dc5edc7bd491",
+      "git-tree": "6dede6ba1ac505ea4bd64c60bf97edfe7b8d5647",
+      "version-date": "2023-01-16",
+      "port-version": 1
+    },
```
Comment (Adela0814) [inline versions/t-/thomasmonkman-filewatch.json:7]: Please restore the original git tree back.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: versions/t-/thomasmonkman-filewatch.json (hunk 1 above)
Comment (Pravila00) [reply versions/t-/thomasmonkman-filewatch.json:7]: Thanks! I've just restored the original git tree back, let me know if you need anything else from my side.

**Review by Adela0814 — APPROVED** [review]

**Comment by Adela0814** [comment]
This patch has been submitted to the upstream and has not responded for more than 2 years.

**Pull request merged**